Dockerfile

```diff
@@ -13,32 +17,28 @@ RUN find packages \! -name "package.json" -mindepth 2 -maxdepth 2 -print | xargs
 
 # Stage 2 - Install dependencies and build packages
 FROM node:18-bookworm-slim AS build
+ARG ENVIRONMENT_CONFIG
 
-# Set Python interpreter for `node-gyp` to use
-ENV PYTHON /usr/bin/python3
-
-# install sqlite3 dependencies
+# Install sqlite3 dependencies. You can skip this if you don't use sqlite3 in the image,
+# in which case you should also move better-sqlite3 to "devDependencies" in package.json.
 RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
     --mount=type=cache,target=/var/lib/apt,sharing=locked \
     apt-get update && \
-    apt-get install -y --no-install-recommends libsqlite3-dev python3 build-essential && \
-    rm -rf /var/lib/apt/lists/*
```

Comment:
Note that deleting the apt lists CAN massively reduce the image layer sizes; i think it's pretty much always recommended to have in place.

Reply:
btw it's kinda interesting that python and build-essentials are not needed anymore; do you have some background for why that was the case?

Reply:
So bit of overzealousness on my part added both back. I think for now we could get away without python and build-essentials due to the fact that we don't have the scaffolder installed but I think keeping it is fine.